backend/src/common/utils/userHelper.ts

```diff
@@ -0,0 +1,7 @@
+import { createToken } from './tokenHelper';
+import { IUser, IUserResponse } from '../models/user';
+
+export const signUpResponseMapper = (user: IUser): IUserResponse => ({
+  ...user,
+  token: createToken({ id: user.id })
+});
```

Comment:
incorrect. rollback it. the previous version was done on purpose. or pass a token here because createToken should happen in service